region/lowernorfair/east.json

```diff
@@ -2655,63 +2716,663 @@
+                      {"and": [
+                        "SpaceJump",
+                        {"heatFrames": 600}
+                      ]},
+                      {"and": [
+                        "SpaceJump",
+                        "HiJump",
+                        {"heatFrames": 390}
+                      ]}
+                    ]}
+                  ]
+                },
+                {
+                  "name": "Amphitheatre Triple SpringBall Jump",
+                  "notable": true,
+                  "requires": [
+                    "h_heatProof",
+                    "canSuitlessLavaDive",
+                    "HiJump",
+                    "canCrouchJump",
+                    "canDoubleSpringBallJumpMidAir",
+                    "canTrickyJump",
+                    {"acidFrames": 330},
+                    {"acidFrames": 1000}
+                  ],
+                  "reusableRoomwideNotable": "Amphitheatre Reverse Acid Dive",
+                  "note": [
+                    "Acid allows for more time than water to perform Spring Ball jumps.",
+                    "This gives just barely enough time to fit in three spring ball jumps."
+                  ],
+                  "devNote": [
+                    "This strat is only used to avoid walljumping.",
+                    "1000 leniency frames for retries, and so this does not become the expected strat"
+                  ]
+                },
+                {
+                  "name": "Amphitheatre Gravity Jump SpringBall Jump",
+                  "notable": true,
+                  "requires": [
+                    "h_canNavigateHeatRooms",
+                    "HiJump",
+                    "canTrickyJump",
+                    "canGravityJump",
+                    "h_canCrouchJumpDownGrab",
+                    "canSpringBallJumpMidAir",
+                    {"heatFrames": 600},
+                    {"acidFrames": 600}
+                  ],
+                  "reusableRoomwideNotable": "Amphitheatre Reverse Acid Dive",
+                  "note": [
+                    "Perform a gravity jump followed by a springball jump to reach the left side door.",
+                    "The gravity jump timing is very precise."
+                  ],
+                  "devNote": [
+                    "This strat is only used to avoid walljumping.",
+                    "FIXME: Gravity heat reduction is disabled for part of this strat."
+                  ]
+                }
+              ]
+            }
+          ]
+        },
+        {
+          "from": 6,
+          "to": [
+            {
+              "id": 1,
+              "strats": [
+                {
+                  "name": "SpinJump Entry",
+                  "notable": false,
+                  "requires": [
+                    "h_canNavigateHeatRooms",
+                    "canPrepareForNextRoom",
+                    {"or": [
+                      "SpaceJump",
+                      "canWalljump"
```

Comment:
```suggestion
                      "canWalljump",
                      "canStationarySpinJump"
```

Reply:
It would be spinjumping forward into the room then turning around.

Reply:
Normal jump entry is harder than I thought, it would be a tricky jump where you  aim down and shoot the open the door.
stationary spinjump like a doorstuck is easier